README.asciidoc

```diff
@@ -0,0 +1,16 @@
+== How to run robot
+
+* Open RobotCode folder

```

Comment:
More precise setup missing. Are there python version dependencies, some libraries you compiled?